Add procedure for refreshing content counts (#2619)
theforeman/foreman-documentation: guides/common/modules/proc_refreshing-smart-proxy-counts.adoc

[id="Refreshing_Smart_Proxy_Counts_{context}"]
= Refreshing {SmartProxy} Counts

If your {SmartProxies} have synchronized content enabled, you can refresh the number of content counts available to the environments associated with the {SmartProxy}.

.Procedure

> . In the {Project} web UI, navigate to *Infrastructure* > *{SmartProxies}*, and select the {SmartProxy} where you want to see the synchronized content.

Reviewer: ```suggestion
. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*, and select the {SmartProxy} where you want to see the synchronized content.
```

> . Select the *Overview* tab and under *Content Sync*, toggle the *Synchronize* button to do an *Optimized Sync* or a *Complete Sync* to synchronize the {SmartProxy} which refreshes the content counts.

Reviewer: ```suggestion
. Select the *Overview* tab.
. Under *Content Sync*, toggle the *Synchronize* button to do an *Optimized Sync* or a *Complete Sync* to synchronize the {SmartProxy} which refreshes the content counts.
```

> . Select the *Content* tab and choose an *Environment* to view Content Views available to those {SmartProxies} by clicking the greater than symbol to expand.

Reviewer: ```suggestion
. Select the *Content* tab.
. Choose an *Environment* to view Content Views available to those {SmartProxies} by clicking *>*.
```

> . Expand the Content View by clicking the greater than symbol to view repositories available to the Content View and the specific version for the environment.

Reviewer: ```suggestion
. Expand the Content View by clicking *>* to view repositories available to the Content View and the specific version for the environment.
```

. View the number of content counts under *Packages* specific to yum repositories. 

> . View the number of files errata, package groups, files, container tags, container manifests, Ansible collections under *Additional content*.

Reviewer: ```suggestion
. View the number of errata, package groups, files, container tags, container manifests, and Ansible collections under *Additional content*.
```
files is twice here.
